libcommon/securestr.c

```diff
@@ -22,16 +22,14 @@
 
 #include <ctype.h>
 #include <stdio.h>
-#include <stdlib.h>
 #include <string.h>
 
-char *ssafe(char *s) {
+char *ssafe(const char *s) {
 	int i = 0,
 	    j = 0;
-	char safe[BUFSIZ] = "",
-	     *ssafe = "";
+	char safe[BUFSIZ] = "";
 
-	for (i = 0; s[i]; i++) {
+	for (i = 0, j = 0; s[i]; i++, j++) {
```

Comment:
This may not be valid according to the language standard. "Two mutations in one expression"

Reply:
Well, it compiles with ``-std=c99`` on GNU's cc, didn't test with clang yet.

Reply:
Yeah, clang is a lot more stubborn with standards and it compiled just fine.
I guess it's actually part of C99.